### ISSUE TITLE
~jcdl

### ISSUE THREAD

**Issue by tg-z** [tg-z/web-clip#890]
~jcdl        

[~jcdl](/)

[RSS](http://jcdl.net/index.xml)

[Noteworthy theme](https://github.com/kimcc/hugo-theme-noteworthy)

[Built with Hugo](https://gohugo.io)

A ThinkPad For 2020
===================

February 10, 2020

[![](http://jcdl.net/images/thinkpad2020/small/2.jpg)
](/images/thinkpad2020/2.jpg)

I’ve always loved ThinkPads. Every time I touch one I feel a pang of regret and jealousy because, for the last few years, I’ve been using MacBooks. I still have my old T410, used to have an X230T, frequently help my mom with her P53, and a couple years ago I bought an old X60T for $50 to play around with. Every time I pull out that X60T I mourn the death of sensible notebook computing. Inspired by the very best from Apple, Lenovo, and others, this is the machine I would immediately trade in my 16” MacBook Pro for. I fully believe the machine described is practical to manufacture today. These ideas are not that out there.

tl;dr is at the bottom of the page.

1\. Trackpoint and trackpad
---------------------------

[![](http://jcdl.net/images/thinkpad2020/small/8.jpg)
](/images/thinkpad2020/8.jpg)

Keep it as is and throw a couple different nipples in the box as spares, or different textures. It would be a crime to ship a ThinkPad without a trackpoint. Please, I beg you, don’t kill it, and keep all three buttons below the spacebar. As for the trackpad and mouse buttons, slightly quieter buttons would be appreciated if they could maintain tactile feedback. They could also be slightly lighter to press, but that’s minor. I’d be okay without physical buttons on the bottom of the trackpad **only if** they were able to copy Apple’s vibration motor technique. Lenovo’s clickpads are absolutely horrid, and it’s a relief that the P53’s trackpad is as good as it is. The tracking accuracy and smooth texture on the P53 is surprisingly good, and the size is good. Obviously Apple’s trackpads are second to none, and should be what every company aims to copy.

[![](http://jcdl.net/images/thinkpad2020/small/1.jpg)
](/images/thinkpad2020/1.jpg)

2\. Keyboard
------------

[![](http://jcdl.net/images/thinkpad2020/small/6.jpg)
](/images/thinkpad2020/6.jpg)

I could ask for a thicker keyboard with one of Kailh’s notebook switches[1](#fn:1), but to keep things reasonable and thin, the current ThinkPad keyboard is acceptable. Must be backlit. I would like to see higher quality plastics used that are less prone to shine over time.

I like both the classic ThinkPad keyboard and the current chicklet style design and I would be satisfied with both. It’d be nice if they brought back the blue Enter, but that’s just nostalgia.

I do not want a numpad; gotta keep that keyboard and trackpad centered. It should also be easily replacable and have liquid spill channels.

3\. Display
-----------

[![](http://jcdl.net/images/thinkpad2020/small/4.jpg)
](/images/thinkpad2020/4.jpg)

The display is by far my biggest ask and the single biggest factor that keeps emptying my wallet at the Apple store. Please, please, please ship a laptop with a 16:10, or better yet, 3:2 display. Just look at the current lineup and how their displays are built. On the P53, there is a full 20mm of unused space below the display, and about 16mm wasted above the display. As it turns out, a 15.0” 16:9 display is just about 20mm shorter than a 15.4” 16:10 display. A 15.7” 3:2 display is 34.5mm taller than a 15.0” 16:9 display, which still fits within the current frame, just barely. Dell can do near borderless displays, why can’t Lenovo? Although I used 15” displays as an example here, I think 14” is the sweet spot for portability, especially if it has super thin bezels.

Resolution, panel type, brightness, and finish are just as important as the aspect ratio.

*   IPS and OLED options should be options (microLED once it’s availble)
*   75Hz, 90Hz, or 120Hz support would be a dream (it’s not just for gamers!)
*   Retina and non-retina resolution options
*   Glossy and matte texture finish glass options (no more bare LCDs)
*   400 nit brightness, bare minimum. No more 250 nit bullshit
*   Touch screen option, for the weird people who like fingerprints on their screens

[![](http://jcdl.net/images/thinkpad2020/small/5.jpg)
](/images/thinkpad2020/5.jpg)

4\. Processor
-------------

This section is easy. Ryzen 4000 7nm series APUs. The 8-core, 16-thread Ryzen 7 4800U would be a great chip for an X1 Carbon-style notebook. The higher TDP Ryzen 7 4800H would be great for a T- or P-series notebook. Buying an Intel CPU at this point means buying old tech, and you may very well be gambling future performance away with the speculative execution bug of the week. I have a very hard time convincing myself that a >65W 14nm++ Comet Lake H chip would be worth it, even if it has 10 cores[2](#fn:2). I know nanometer numbers are just marketing, but 7nm is the smarter choice, and AMD’s integrated graphics are better.

5\. Graphics
------------

The integrated graphics with the Ryzen APUs would be sufficient for most tasks, and I would appreciate the option to buy a notebook without a dedicated GPU to save a bit. I personally don’t care for Nvidia as a company, as it eliminates any hope of hackintosh compatibility at this point, though professionals would certainly appreciate Quadro GPUs. I’m happy with the AMD Navi graphics in my current MacBook Pro 16”, so that would be fine for me.

6\. Memory
----------

Easily expandable and fast. This is classic ThinkPad. Nothing soldered on bigger models, please. ECC should also be possible with Ryzen 4000, as it is on their desktop parts, but I’m not sure about that, and it doesn’t really matter to me.

7\. Storage
-----------

PCIe 4.0 NVMe, two slots. I don’t care about SATA on smaller models, give me more battery space instead.

8\. Battery
-----------

What ever happened to externally removable batteries? I’ve had some less than fun experiences with battery clips coming loose in my T410. If they’re going to be internal batteries, please don’t glue them down and allow them to be user servicable. I like the multi-battery approach with the smaller internal one to keep the machine running during a swap.

Anything less than 80Wh total in a T-series laptop is unacceptable at this point, in my opinion. I’d love to see 99.9Wh batteries in a P-series.

9\. I/O
-------

[![](http://jcdl.net/images/thinkpad2020/small/7.jpg)
](/images/thinkpad2020/7.jpg)

Thunderbolt 3.0 with USB-C charging, at least two ports. At least two type-A USB-3.whatever ports. SD card slot. RJ-45 Ethernet (a fold out port is fine), 10Gig would be a lovely touch. HDMI. WiFi 6 via mPCIe. 4K webcam, I’ll do without the shutter if it means thinner bezels. IR login camera. Smartcard slot. LTE as an option. Fingerprint reader. NFC. Good speakers and a good mic[3](#fn:3). Just keep doing what you’re doing, Lenovo! Just keep the ports off the back. They’re hard to reach. Put cooling vents there instead.

10\. OS
-------

First class Linux support for all the hardware and power management, as long as it targets a recent kernel. Whatever the current Ubuntu LTS is is fine. FreeBSD support would be a game changer too, however unlikely. And please let me buy it without a Windows license.

11\. Other stuff
----------------

[![](http://jcdl.net/images/thinkpad2020/small/0.jpg)
](/images/thinkpad2020/0.jpg)

No coil whine. No touch bar. Lenovo does a good job making laptops as thick as they need to be, so keep doing that. Make it as thick as it needs to be to keep things cool and near silent (a good reason on its own to avoid discrete GPUs). I cautiously welcome the idea of a separate low-power co-processor for security, as long as it has open source drivers and allows me to use generic components. Would factory Libreboot or Coreboot support be too much to ask?[4](#fn:4) No Chinese spyware either please.

Conclusion
----------

Lenovo, I know you can do this. The ThinkPad 25 would have been nearly perfect if you just gave it a 16:10 display. You can do this. Do it for me. Oh and throw in a slide out secondary display[5](#fn:5).

TL;DR: I just want a repairable T495S with 7nm Ryzen with a taller and brighter display that feels like a classic ThinkPad.

[![](http://jcdl.net/images/thinkpad2020/small/3.jpg)
](/images/thinkpad2020/3.jpg)

* * *

1.  Kailh is a popular mechanical keyboard switch manufacturer, mostly popular for the Cherry MX-style switches. They also make some low profile switches that [could be used in a notebook](http://www.kailh.com/en/Products/Ks/NotebookS/649.html). [↩︎](#fnref:1)
    
2.  A [leaked roadmap](https://wccftech.com/intel-desktop-mobile-cpu-roadmap-leak-14nm-comet-lake-10nm-ice-lake-tiger-lake/) shows 10nm is not coming soon enough. [↩︎](#fnref:2)
    
3.  It’s really amazing how good the 16” MacBook Pro’s speakers and mic are. They kept the same absolute shite webcam, for some reason. [↩︎](#fnref:3)
    
4.  Yes. [↩︎](#fnref:4)
    
5.  [Just kidding.](https://www.lenovo.com/us/en/laptops/thinkpad/w-series/w700ds/) [↩︎](#fnref:5)
    

* * *

[Noteworthy theme](https://github.com/kimcc/hugo-theme-noteworthy)

|

[Built with Hugo](https://gohugo.io) 
 http://jcdl.net/posts/thinkpad-2020/
